### ISSUE TITLE
Carousel controls overflow in home page

### ISSUE THREAD

**Issue by arielj** [rubyforgood/rubyforgood.org#19]
The controls of the carousel at the bottom of the page are placed outside the container, adding extra width to the document and adding that extra scroll to the right side.

![rubyforgood org_(iPhone X) (1)](https://user-images.githubusercontent.com/188464/130542734-31feeeb8-ee8b-401c-ba86-86d999e879a9.png)

Some options to fix this:
1. hide the controls for the resolution when they start to not fit into the width
2. or, resize the container of the carousel so the controls are still inside the viewport

I think option 1 is simpler since on those devices the user can move the carousel swiping with the thouch

**Comment by meg-gutshall** [comment]
Hi Graziele! Thank you for jumping in and volunteering!

I will assign you this issue, but I'm going to mark it as blocked for now. Tonight I will go on and make sure everything is up-to-date on here (I've been focusing on Human Essentials lately, so I have to check up on the status of this) and then unblock it when it's ready for contribution!

Does that sound good to you? Please let me know if you have any questions!

**Comment by meg-gutshall** [comment]
Hey @grazirs! Thank you for your contribution! It looks good, but what I _really_ want to do in the long run is rip out OwlCarousel and Slick and replace them with [Flickity](https://flickity.metafizzy.co/). Would you be up to taking on that task? I know it's a bit large, but I'm being pulled in a lot of different directions and haven't been able to make as much progress on this site as I would have liked.

If not, that's fine and I can merge in your PR as is. Just let me know!

**Comment by grazirs** [comment]
Hey @meg-gutshall, sure! I can work on it. Is the goal to reproduce the same design and behaviors with Flickity?

**Comment by meg-gutshall** [comment]
That's correct! Right now, they're all shoved into `assets/js/main.js`. It would be nice to have them cleaned up and separated out a bit. I'll write up a new issue real quick and assign it to you. Thanks a ton!

**Comment by meg-gutshall** [comment]
Hi @grazirs, I created the new issue #54 this afternoon, but I think you have to comment on it before I can assign it to you since you're not listed as a member in the Ruby for Good GitHub organization.
